src/App.tsx

```diff
@@ -15,35 +16,97 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortField {
+  Name = 'name',
+  Length = 'length',
+  NoField = '',
+}
+
+type SortProps = {
+  sortBy: SortField;
+  reversed: boolean;
+};
+
+function getPreparedGoods(products: string[], { sortBy, reversed }: SortProps) {
```

Comment:
Make sense to create different TS file and put the function in it.

Reply:
Agreed! Please, move func to a separate file.